### PR TITLE
Issue #103 : attempt to improve the top menu.

### PR DESCRIPTION
**Pull request by giant-teapot**
This commit adds FontAwesome (v4.2.0) to the lib folder.

### CODE REVIEW

**Comment by giant-teapot** [comment]
There is still room for improvement (top right menu for instance, colors and so on), but I'd really like feedback before putting more work into this.

**Comment by Yoric** [comment]
Have you tested on a mobile device?

**Comment by giant-teapot** [comment]
Tested and kinda fixed. Should display nicely for everyone (hopefully).

File: css/menu.css [hunk 1]
```diff
@@ -30,6 +31,10 @@
 .top {
   position: absolute;
   top: 0;
+
+  /*padding: 150px 10px;*/
```
Comment (Yoric) [inline css/menu.css]: Please don't leave code commented out.

File: css/menu.css [hunk 2]
```diff
@@ -44,25 +49,103 @@
   cursor: pointer;
 }
 
+#menu_top_contents {
+  width: 50%;
+  margin: auto;
+  min-height: 45px;
```
Comment (Yoric) [inline css/menu.css]: What does this `min-height` map to? Is it the size of images? If so, could you add a comment?

File: css/menu.css [hunk 3]
```diff
@@ -44,25 +49,103 @@
...
+  font-weight: bold;
+}
+
+#menu_top_contents p {
+  margin: 0px 3px;
+}
+
+#menu_top_left {
+  height:30px;
```
Comment (Yoric) [inline css/menu.css]: Same here, where do these numbers come from?

File: css/menu.css [hunk 4]
```diff
@@ -44,25 +49,103 @@
...
+  width: 30px;
+  margin-top: 8px;
+  margin-left: 15px;
+  background: linear-gradient(to bottom, #20A420, darkgreen);
```
Comment (Yoric) [inline css/menu.css]: Nit: Not compulsory, but I personally find `rgb(...)` more readable than hex.

File: css/menu.css [hunk 5]
```diff
@@ -77,10 +160,12 @@
 
 .folded_up {
   height: 0;
+  opacity: 0;
 }
 
 .unfolded_up {
-  height: 3em;
+  height: 60px;
```
Comment (Yoric) [inline css/menu.css]: Again here, where do these `60px` come from?

File: lib/font-awesome-4.2.0/css/font-awesome.css [hunk 6]
```diff
@@ -0,0 +1,1672 @@
+/*!
```
Comment (Yoric) [inline lib/font-awesome-4.2.0/css/font-awesome.css]: Could you make this a submodule instead of a hard-coded copy of a specific version of the code?

**Comment by Yoric** [comment]
Thanks for the patch. I'd like a few changes before I merge it.

File: css/menu.css (hunk 3 above)
Comment (giant-teapot) [inline css/menu.css]: Tried to address those ugly magic numbers with 4dab7a2 
The only remaining should be the menu button dimensions (35x35 pixels). Should I leave a comment about this ?

Thanks for your thorough review, tell me if I missed something.

**Comment by Yoric** [comment]
Also, note that the pull request cannot be merged at the moment. You should `git pull --rebase upstream` and resolve any conflicts before you proceed.

**Comment by Yoric** [comment]
Note that I still can't pull. Don't forget to rebase.

**Comment by Yoric** [comment]
Since these changes cannot be imported independently, could you fold this to a single commit?

File: _config.yml [hunk 7]
```diff
@@ -0,0 +1,5 @@
+## Jekyll/Githup pages configuration
+
+# Ignore Font-Awesome `src` folder which is the source repository for the lib's
```
Comment (Yoric) [inline _config.yml:3]: Could you document why?

**Comment by Yoric** [comment]
Let's ship it!

**Pull request merged**